### PR TITLE
Update locators

### PR DESCRIPTION
**Pull request by shweta83**
Updated Search Locator and Locator for PopOver Widget

### CODE REVIEW

File: airgun/widgets.py [hunk 1]
```diff
@@ -635,7 +635,7 @@ class Search(Widget):
     )
     search_field = TextInput(
         locator=(
-            ".//input[@id='search' or contains(@placeholder, 'Filter') or "
+            ".//input[@id='search' or contains(@placeholder, 'Search') or "
             "@ng-model='table.searchTerm' or contains(@ng-model, 'Filter') or "
```
Comment (jameerpathan111) [inline airgun/widgets.py:639]: @shweta83 this change is not required, see line number 640. It already covers it(`contains(@placeholder, 'Search')`).

**Review by jameerpathan111 — CHANGES_REQUESTED** [review]

File: airgun/widgets.py (hunk 1 above)
Comment (shweta83) [reply airgun/widgets.py]: Yes. I had old airgun so It was wasn't there. I undo that.

**Comment by latran** [comment]
@shweta83 is there a particular test case that you can associate with this.  I'm not sure the change surrounding this PR without a good context of where its usage is.

**Comment by shweta83** [comment]
> @shweta83 is there a particular test case that you can associate with this. I'm not sure the change surrounding this PR without a good context of where its usage is.

@latran All the UI tests for settings component access settings page in Administer -> Settings and when we change any setting, popover widget opens.  
https://github.com/SatelliteQE/robottelo/blob/b0e7f8fe798cdd4f93f6c16237a4f0167a850dd6/tests/foreman/ui/test_settings.py#L75

**Review by latran — APPROVED** [review]
ACK

File: airgun/widgets.py [hunk 2]
```diff
@@ -2181,7 +2181,7 @@ class PopOverWidget(View):
     """
 
     ROOT = '.'
-    column_value = Text(".//span[contains(@class, 'editable')]")
+    column_value = Text(".//div[@class='ellipsis editable']")
```
Comment (omkarkhatavkar) [inline airgun/widgets.py:2184]: image

A Better Locator would be using contains as there are empty ones. e.g. $x("//div[contains(@class,'ellipsis editable')]")

**Review by omkarkhatavkar — APPROVED** [review]
ACK

**Review by omkarkhatavkar — APPROVED** [review]

**Pull request merged**